lib/ramfs/ramfs.h

```diff
@@ -36,29 +36,65 @@
 #include <vfscore/prex.h>
 #include <stdbool.h>
 
-/*
- * File/directory node for RAMFS
+/**
```

Comment:
Do not use dots (`.`) at the end of a comment line, unless it is a sentence. Do start comments with a capital, though.